File: thats-odd/ThatsOdd.c
#include <stdio.h>

int main() {
  int count;
  scanf("%d", &count);
  
  int summa = 0;
  while (count--) {
    int number;
    scanf("%d", &number);
    if (number % 2 == 0) summa += number;
  }
   
  printf("%d", summa);
  return 0;
}

File: thats-odd/ThatsOdd.java
import java.util.Scanner;

public class Program
{ 
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int count = input.nextInt();
        int summa = 0;
        
        while (count-- > 0) {
            final int number = input.nextInt();
            if (number % 2 == 0) {
            	   summa += number;
            }
        }
        System.out.print(summa);
    }
}
